datafusion/core/src/physical_plan/filter.rs

```diff
@@ -197,7 +210,14 @@ impl ExecutionPlan for FilterExec {
         let input_stats = self.input.statistics();
         let input_column_stats = match input_stats.column_statistics {
             Some(stats) => stats,
-            None => return Statistics::default(),
+            None => self
```

Comment:
this seems like it is more general than just for `FilterExec` -- shouldn't all ExecutionPlan's return statistics that have unbounded columns in the absence of other information? Maybe we should change `Statistics::default()` to do this 🤔

Reply:
I agree with you.  We can remove `Statistics::default()` implementation, and propagate unbounded columns (in the absence of information) from the source. I will try it.